src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -0,0 +1,273 @@
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-help-1.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-help-2.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+		<div class="sky-subsection-heading">Modal dialog behavior</div>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>When users perform actions in the flyout to launch modal dialogs, the modal dialogs should cover the entire page.</p>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/ixn-modal.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>
+
+	<header><div class="sky-section-heading">Options</div></header>
+		<p class="sky-text-warning">This section describes future options for the flyout component. The component remains in development and the options described in this section are not yet available.</p>
+		<div class="sky-subsection-heading">Next/previous buttons</div>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>When a flyout opens records in a list, you can include buttons in the toolbar to open the next record or previous record. For example, if users open a flyout on a constituent record to view the first gift in a list of gifts, the next button opens the second gift in the list. When you include the next and previous buttons in the flyout toolbar, a visual indicator highlights the currently selected row on the page.</p>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/opt-arrows.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>		
+	
+		<div class="sky-subsection-heading">List toolbar</div>
+			<stache-row>
+				<stache-column screenSmall="6">
+					<p>When a flyout displays a list, you can include an option in the toolbar to leave the "preview" list and create a new list for further manipulation. The create list button navigates to a new page that contains the new list.</p>
+				</stache-column>
+				<stache-column screenSmall="6">
+					<div class="screenshot screenshot-info">
+						<img src="~/assets/img/guidelines/flyout/opt-list.png" style="width: 100%;" />
+					</div>
+				</stache-column>
+			</stache-row>		
+
+	<header><div class="sky-section-heading">Responsiveness</div></header>
+		<div class="sky-subsection-heading">Mobile behavior</div>
+			<stache-row>
+    		<stache-column screenSmall="6">
+					<p>Do not use a flyout when a screen is smaller than 480px. At smaller resolutions, the flyout covers too much of the screen and loses much of its value. For smaller screen sizes, navigate directly to the appropriate record or list.</p>
```

Comment:
Dow we need the screenshot with this item? I don't see how it really adds anything to the description of this scenario.